src/hooks/useKonamiCode/KonamiModal.tsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable jsx-a11y/media-has-caption */
+import { useRef, useState } from "react";
+import { css } from "@emotion/react";
+import MarketingModal from "@leafygreen-ui/marketing-modal";
+import { InlineKeyCode } from "@leafygreen-ui/typography";
+import Cookies from "js-cookie";
+import { SEEN_KONAMI_CODE } from "constants/cookies";
+import { zIndex } from "constants/tokens";
+import useKonamiCode from ".";
+import evergreenKonami from "./EvergreenKonami.png";
+import streetfighterTree from "./StreetFighterTree.png";
+
+const KonamiModal = () => {
+  const [open, setOpen] = useState(Cookies.get(SEEN_KONAMI_CODE) !== "true");
```

Comment:
Nitpicking but would be cool if there was a delay before the modal appeared to emphasize the before/after effects.

Reply:
Set the delay for 2 seconds